npm/webpack-preprocessor/.gitignore

```diff
@@ -4,5 +4,4 @@ node_modules/
 _test-output
 **/cypress/videos
 cypress/screenshots
-examples/react-app/build/
```

Comment:
We might want to keep these .gitingnore paths around? Stale branches pulling in this change will likely might have these git ignore files laying around and git will suggest we check them in?

Reply:
I'll revert! Not harming anyone and less confusion for devs.